Sources/AppStoreConnectCLI/Commands/TestFlight/PreReleaseVersion/ListPreReleaseVersionsCommand.swift

```diff
@@ -0,0 +1,92 @@
+// Copyright 2020 Itty Bitty Apps Pty Ltd
+
+import AppStoreConnect_Swift_SDK
+import ArgumentParser
+import Foundation
+
+struct ListPreReleaseVersionsCommand: CommonParsableCommand {
+    static var configuration = CommandConfiguration(
+        commandName: "list",
+        abstract: "Get a list of prerelease versions for all apps.")
+
+    struct Options: ParsableArguments {
```

Comment:
🤔 I'm seeing some commonality between some of these arguments and some arguments in other commands

I think we should maybe chunk option groups together to share them between commands, especially to share the text

I know you were trying to use this group to translate into the operation options. But I think they can have different purposes